sentry-kotlin-multiplatform/src/androidUnitTest/kotlin/io/sentry/kotlin/multiplatform/ContextProviderTest.kt

```diff
@@ -0,0 +1,45 @@
+package io.sentry.kotlin.multiplatform
+
+import android.content.ContentValues
+import android.net.Uri
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+
+@RunWith(AndroidJUnit4::class)
+class ContextProviderTest : BaseSentryTest() {
```

Comment:
`l`: Maybe add one test to verify the `context` is non-null after `onCreate` is called?